src/meta-client/src/client/cluster.rs

```diff
@@ -72,27 +71,10 @@ impl Client {
         let inner = self.inner.read().await;
         inner.batch_get(req).await
     }
-}
-
-#[async_trait::async_trait]
-impl ClusterInfo for Client {
-    type Error = Error;
-
-    async fn list_nodes(&self, role: Option<cluster::Role>) -> Result<Vec<NodeInfo>> {
-        let cluster_id = self.inner.read().await.id.0;
-        let key_prefix = match role {
-            None => NodeInfoKey::key_prefix_with_cluster_id(cluster_id),
-            Some(role) => NodeInfoKey::key_prefix_with_role(cluster_id, role),
-        };
-
-        let req = RangeRequest::new().with_prefix(key_prefix);
-
-        let res = self.range(req).await?;
 
```

Comment:
Let's add some docs for this pub functions, looks like it returns the metasrv leaders and followers.

Reply:
This method is just `range`, it doesn't have any further significance.